metron-deployment/packaging/ambari/metron-mpack/src/main/resources/common-services/METRON/CURRENT/package/scripts/indexing_master.py

```diff
@@ -119,6 +119,10 @@ def start(self, env, upgrade_type=None):
                 msg = "WARNING: Elasticsearch index templates could not be installed.  " \
```

Comment:
I thought it would be better to throw the exception and let the caller handling exception based on wether to continue further or not rather handling the exception inside the method. 
I can still handle the exception inside the method and make the method return the status of the installation of template ?